### PR TITLE
Annotate Bool and Number literals

### LINKED ISSUE #116: NumberLiteral doesn't return token
Test script:
```go
package main

import (
	"fmt"

	bblfsh "gopkg.in/bblfsh/client-go.v3"
	"gopkg.in/bblfsh/client-go.v3/tools"
	"gopkg.in/bblfsh/sdk.v2/uast"
	"gopkg.in/bblfsh/sdk.v2/uast/nodes"
)

var content = `public class Test {
  public String foo(int i) {
    if (i == 0) return "Foo!";
  }
}
`

func main() {
	client, err := bblfsh.NewClient("0.0.0.0:9432")
	if err != nil {
		panic(err)
	}

	res, _, err := client.
		NewParseRequest().
		Mode(bblfsh.Annotated).
		Language("java").
		Content(content).
		UAST()
	if err != nil {
		panic(err)
	}

	query := "//*[@role='Literal']"
	it, _ := tools.Filter(res, query)
	for it.Next() {
		n := it.Node().(nodes.Node)
		fmt.Printf("Type: %s, Token: %s\n", uast.TypeOf(n), uast.TokenOf(n))
	}
}
```

Output:
```
$ go run uast/issue/issue.go
Type: NumberLiteral, Token:
Type: StringLiteral, Token: "Foo!"
```

But it has token which is visible in bblfsh/web:
Screenshot 2019-04-16 at 13 37 47

### PR DESCRIPTION
**Pull request by bzz**
Fixes #116 by annotating 2 type of literals, so they would have `@token` field.

### CODE REVIEW

**Comment by dennwc** [comment]
CI fails. It probably needs a rebase as well.

**Comment by bzz** [comment]
> CI fails. It probably needs a rebase as well.

CI failure seems to be the rebased artefact. Let me fix it

**Comment by bzz** [comment]
@dennwc rebase artefacts removed and CI is green now. Ready for another round.

File: driver/normalizer/annotation.go [hunk 1]
```diff
@@ -28,6 +28,8 @@ var Native = Transformers([][]Transformer{
 	},
 }...)
 
+var Code []CodeTransformer // TODO(bzz): deprecated
```
Comment (dennwc) [inline driver/normalizer/annotation.go]: This was removed during SDK update as deprecated.

Comment (bzz) [reply driver/normalizer/annotation.go]: Thank you for spotting! A rebase went rogue...

**Comment by bzz** [comment]
Feedback addressed by re-writing the history in c13bfe0. Thank you @dennwc !

**Comment by bzz** [comment]
Thanks for the prompt reviews! Will merged as soon as CI is green

**Pull request merged**